src/main/java/ca/gc/cyber/ops/assemblyline/java/client/clients/ProxyProperties.java

```diff
@@ -0,0 +1,19 @@
+package ca.gc.cyber.ops.assemblyline.java.client.clients;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Data
+@ConfigurationProperties(prefix = "assemblyline-java-client.proxy")
+public class ProxyProperties {
+
+    /**
+     * Proxy host.
+     */
+    private String host;
+
+    /**
+     * Proxy port.
+     */
+    private int port;
```

Comment:
sorry my last comment was incorrect, this should actually be Integer and then I think you can use @Min and null

Reply:
If using int, by default port will be set to 0, Integer will allow for null

Reply:
Changed for an Integer